modules/arriba/main.nf

```diff
@@ -0,0 +1,54 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+// TOOL DESCRIPTION:
+// Arriba is a command-line tool for the detection of gene fusions from RNA-Seq data. It was developed for the use in a clinical research setting.
+// Apart from gene fusions, Arriba can detect other structural rearrangements with potential clinical relevance, such as viral integration sites,
+// internal tandem duplications, whole exon duplications, truncations of genes (i.e., breakpoints in introns and intergenic regions).
+// Source: https://github.com/suhrig/arriba
+// Article: Sebastian Uhrig, Julia Ellermann, Tatjana Walther, Pauline Burkhardt, Martina Fröhlich, Barbara Hutter, Umut H. Toprak, Olaf Neumann, Albrecht Stenzinger, Claudia Scholl, Stefan Fröhling and Benedikt Brors: Accurate and efficient detection of gene fusions from RNA sequencing data. Genome Research.
+// DOI: 10.1101/gr.257246.119
```

Comment:
This should all be moved to `meta.yml` where appropriate.

Reply:
Done!